app/styles/_overview.less

```diff
@@ -272,8 +269,16 @@
     border-top-color: @list-pf-border-color;
     &.active {
       border-top-color: @list-pf-item-active-border-color;
-      @media (min-width: @screen-xs-min) {
-        overflow: hidden; // so animation offsets are hidden when sliding in horizontally
+    }
+    &.deployment-in-progress {
+      // So animation offsets are hidden when sliding in horizontally
+      // Only use this style when a deployment is in progress so that chart
+      // tooltips aren't clipped.
+      overflow:hidden;
+      .list-pf-expansion {
+        @media (max-width: @screen-xs-max) {
+          overflow: hidden; // Hidden when sliding in vertically
```

Comment:
Can we get away with only an `overflow: hidden` on `.list-pf-expansion`? It's not clear to me why it's needed on `.list-pf-item`